src/Sylius/Component/Core/OrderProcessing/StateResolver.php

```diff
@@ -86,11 +86,8 @@ protected function getShippingState(OrderInterface $order)
         $states = array_unique($states);
 
         $acceptableStates = [
```

Comment:
All code related to shipments should be removed from here.

Btw. We should remove this class, is it used anywhere?